deploy/osd-project-request-template/02-clusterrole.dedicated-admins-project-request.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dedicated-admins-project-request
+rules:
+- apiGroups:
+  - template.openshift.io
+  resources:
+  - templates
+  resourceNames:
+  - project-request
+  verbs:
+  - get
+  - list
```

Comment:
Is it possible to list on a single object?

Reply:
I've only ever used a `list` in operator code... it doesn't seem to be possible from the CLI. I'll remove it since it's probably not necessary.